LeaderboardBackend/Services/Impl/ModshipService.cs

```diff
@@ -30,8 +30,14 @@ public async Task<List<Modship>> LoadUserModships(Guid userId)
 					.ToListAsync();
 	}
 
-	// TODO: Implement this
-	// public async Task DeleteModship(Modship modship)
-	// {
-	// }
+	public async Task DeleteModship(Modship modship)
+	{
+		var deleteModship = _applicationContext.Modships.Where(m =>
```

Comment:
Not your fault because we forgot to write down in our code style but we're opting not to use `var`. Would you be able to change this to a `Modship?`?

Reply:
Probably obsoleted by new Requested Change